src/Microsoft.ApplicationInsights/Extensibility/Implementation/Endpoints/ConnectionStringDuplicateKeyException.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Endpoints
+{
+    using System;
+#if !NETSTANDARD1_3
+    using System.Runtime.Serialization;
+#endif
+
+    /// <summary>
+    /// This exception is used to notify the user that their Connection String has duplicate keys.
+    /// </summary>
+#if !NETSTANDARD1_3
+    [Serializable]
```

Comment:
why should they be serializable?

Reply:
FxCop insisted on this. Our other custom exceptions have the same thing.

Reply:
The one recommendation I found on MSDN is [this](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/ms229064(v=vs.100)) "Do make exceptions serializable. An exception must be serializable to work correctly across application domain and remoting boundaries." 

which makes no sense - we do not cross-app-domian and I hope nobody does remoting in 2019.

But I guess this is fine to keep it for now and consider relaxing out fxcop rules so it does not require us to do meaningless things :)